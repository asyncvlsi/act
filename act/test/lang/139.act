deftype rec (int a, b) { }

defproc test()
{
  chan(rec) a;
  chan(int) x;

  dataflow {
    x -> a.a
  }
}

test t;
