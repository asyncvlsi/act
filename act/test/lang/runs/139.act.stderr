ERROR: File `139.act', line 9, col 6
	Identifier on the RHS of a dataflow expression must be of channel type
   ID: a.a
